BUILD/equipment/off-hand.dat

```diff
@@ -26,6 +26,7 @@ Keg Shield
 Wicker Shield
 Sawblade Shield
 Yorick
+Red X Shield
```

Comment:
You need to also run `assemble.sh` from in the `BUILD` directory and add `RELEASE/data/sl_ascend_equipment.txt` to make this actually apply. If you can't run bash scripts, I can just do it after I merge this pull request.

Reply:
Also done. I wasn't sure if this was something I had to do or would be done in the repo but I'll remember for future changes :+1: